### DISCUSSION TITLE
Request: adjustable fixed increments with "two's complement" instead of "resolution"

### DISCUSSION THREAD

**Discussion by katerlouis** [rsjaffe/MIDI2LR#688] [Ideas]
I personally don't understand how resolution affects the outcome and no matter what number I put it, it's rather unpredictable what happens. Tint for example at a resolution of 512 sometimes gets incremented (per knob click) by 1, by 2 or sometimes even 3. I'd like to have it more predictable, hence the request to add a "increment by" where I can add `1` for Tint and `0,10` for Exposure etc.

What do you think?

**Comment by rsjaffe** [comment]
It's rounding off. The problem with "increment by" is that that is related to the range of the adjustment for the particular photo, and for some adjustments (e.g., exposure), it could be dramatically different depending upon the file format.
